contracts/modules/capital/PriceFeedOracle.sol

```diff
@@ -10,16 +10,22 @@ contract PriceFeedOracle is IPriceFeedOracle {
   mapping(address => OracleAsset) public assets;
 
   address public constant ETH = 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
+  address public immutable safeTracker;
 
   constructor(
     address[] memory _assetAddresses,
     address[] memory _assetAggregators,
-    uint8[] memory _assetDecimals
+    uint8[] memory _assetDecimals,
+    address _safeTracker
   ) {
     require(
       _assetAddresses.length == _assetAggregators.length && _assetAggregators.length == _assetDecimals.length,
       "PriceFeedOracle: different args length"
     );
+    require(_safeTracker != address(0), "PriceFeedOracle: safeTracker cannot be zero address");
+
+    safeTracker = _safeTracker;
+    assets[_safeTracker] = OracleAsset(Aggregator(_safeTracker), 18);
```

Comment:
Given this, we might want to actually implement a dummy `latestAnswer()` function that would return `1e18`. It should be never called, but if it's accidentally called it would return a `1:1` rate.